edb/lib/std/25-numoperators.eql

```diff
@@ -24,6 +24,14 @@
 # for each permutation of common integer and floating-point
 # operand types to avoid casting overhead, as these operations
 # are very common.
+#
+# Our implicit casts do not coincide with PostgreSQL. In particular we
+# do not implicitly cast floats into decimals. The philosophy behind
+# that is that using decimal arithmetic should be opt-in. One of the
```

Comment:
"decimal arithmetic should be opt-in." and that once you're doing decimal arithmetic you won't accidentally switch to doing floating point arithmetic.

Reply:
Added the clarification that decimals should not be accidentally switched back to floats.